ws-nextjs-app/pages/[service]/send/[id]/FormContext/index.tsx

```diff
@@ -91,19 +96,40 @@ export const FormContextProvider = ({
 
     const formData = new FormData();
 
-    // TODO: This is a mock data, we should use the formState instead
-    Object.entries({ surname: 'BBC TEST NAME' }).forEach(([key, value]) => {
-      formData.append(key, value);
+    Object.entries(formState).forEach(([key, item]) => {
+      const fieldValue = item.value;
+
+      if (fieldValue === '') return;
+      if (fieldValue instanceof FileList) {
+        const fileList = fieldValue;
+        const fileListLength = fileList.length;
+
+        for (let fileIndex = 0; fileIndex < fileListLength; fileIndex += 1) {
+          const file = fileList.item(fileIndex);
+          if (file) formData.append(key, file);
+        }
+        return;
+      }
+      if (typeof fieldValue === 'boolean') {
+        if (fieldValue) formData.append(key, 'true');
+        return;
+      }
+      formData.append(key, fieldValue);
     });
 
     try {
-      const url = `https://www.bbc.com/ugc/send/${id}?said=${uuid()}`;
+      const url = `https://www.test.bbc.com/ugc/send/${id}?said=${uuid()}`;
```

Comment:
Should we put this into an environment variable, so that it can be different for different environments?

Reply:
Yeah good shout, I think for now we'll leave this as it's a spike prototype. But when we productionise it, we'll add it, i'll make sure a note is added in the follow up ticket